src/styles/theme.ts

```diff
@@ -92,6 +103,25 @@ export const theme = createTheme({
             background: colors.black50,
           },
         },
+        containedSecondary: {
+          background: '#736BCE1A',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          justifyContent: 'center',
+          border: '10px solid',
+          borderImageSlice: '1',
+          borderWidth: '1px',
```

Comment:
setting `borderWidth` will override the `10px` in  `border: 10px solid`
since that's no longer needed you can just write it as `border: 1px solid`